### PR TITLE
Javascript/javascript3/week2

### PR DESCRIPTION
**Pull request by arya56**

### CODE REVIEW

**Comment by bashirmja** [comment]
Hey Arya, 
As I mentioned before in other code reviews, Please consider removing nonrelated files to this week's PR. I will review your code when you have done your file removal.

**how to:**
check out your branch and remove those file in your solution explorer (file structure) which is not related to this branch.

Best regards,
Bashir

**Comment by arya56** [comment]
> Hey Arya,
> As I mentioned before in other code reviews, Please consider removing nonrelated files to this week's PR. I will review your code when you have done your file removal.
> 
> **how to:**
> check out your branch and remove those files in your solution explorer (file structure) which is not related to this branch.
> then push your changes.
> 
> Hope this could help.
> 
> Best regards,
> Bashir

Hi Bashir,, My files in this branch are all related to the homework! here is the source of homework : https://github.com/HackYourFuture-CPH/JavaScript/blob/master/javascript3/week2/homework.md

Best regards,
Ali

**Comment by bashirmja** [comment]
You are awesome. great job of removing files.

File: javascript/javascript3/week2/badMovies.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const movie_Api = 'https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json'
+const badTitlesAfter2000 = []
+let ali = []
```
Comment (bashirmja) [inline javascript/javascript3/week2/badMovies.js:3]: It is not clear what is this for. consider removing if it is not used or rename it.

File: javascript/javascript3/week2/badMovies.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+async function badMovies(){
```
Comment (bashirmja) [inline javascript/javascript3/week2/badMovies.js:5]: function name shoudl start with a verb to describe the behaivure of the function.
for example:
findBadMovies(), 
getBadMovies(), 
UpdateBadMovies(), 
...

File: javascript/javascript3/week2/badMovies.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    try{
+            const movieResponse = await fetch(movie_Api)
+            const movies = await movieResponse.json()
+
+            const badRated =  movies.filter(movie=>movie.rating < 3)
```
Comment (bashirmja) [inline javascript/javascript3/week2/badMovies.js:10]: Great usage of filter.

**Review by bashirmja — APPROVED** [review]
Approved. Great job. I just made small suggestions to consider them for better coding.

**Pull request merged**